ngraph/core/src/function.cpp

```diff
@@ -405,4 +405,18 @@ void Function::remove_result(const std::shared_ptr<op::Result>& result)
         m_results.end());
 }
 
+void Function::add_parameters(const ParameterVector& params)
+{
+    m_parameters.insert(m_parameters.end(), params.begin(), params.end());
```

Comment:
I think we need to check that new Parameters are not equal to existing ones.

Reply:
added